File: Python/Day2/task4.py
def  checkMilesPerGallon( distanceToCOver = 50, distance1GallonFuelCanCover = 15, fuelRemainInTank= 2):
	let product = fuelRemainInTank * distance1GallonFuelCanCover;
	if(product < distanceToCOver):
		return false;
	else:
		 return true
		
	
result = checkMilesPerGallon();
print(result)

File: JavaScript/Day2/Task4.js
function  checkMilesPerGallon( distanceToCOver = 50, distance1GallonFuelCanCover = 25, fuelRemainInTank= 2){
	
		
	let product = fuelRemainInTank * distance1GallonFuelCanCover;
	if(product < distanceToCOver){
		return false;
	}
	
	
	else return true;
		
	
	}
	
let result = checkMilesPerGallon();
console.log(result)






